.yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs

```diff
@@ -0,0 +1,28 @@
+/*!
```

Comment:
Is this code here meant to be human readable? We might want to break it up a bit.

Reply:
I think this is the module cache